cmd/crank/xpkg/build.go

```diff
@@ -93,48 +93,53 @@ func (c *buildCmd) AfterApply() error {
 
 // buildCmd builds a crossplane package.
 type buildCmd struct {
+	// Flags. Keep sorted alphabetically.
+	EmbedRuntimeImageName    string   `placeholder:"NAME" help:"The name of an OCI image to embed in the package as its runtime." xor:"runtime-image"`
+	EmbedRuntimeImageTarball string   `placeholder:"PATH" type:"existingfile" help:"The tarball of an OCI image to embed in the package as its runtime." xor:"runtime-image"`
+	ExamplesRoot             string   `short:"e" type:"path" help:"A directory of example YAML files to include in the package." default:"./examples"`
+	Ignore                   []string `placeholder:"PATH" help:"Comma-separated paths, specified relative to --package-root, to exclude from the package."`
+	Output                   string   `short:"o" type:"path" placeholder:"PATH" help:"The path to write the built xpkg file to."`
+	PackageRoot              string   `short:"f" type:"existingdir" help:"The directory that contains the package's crossplane.yaml file." default:"."`
```

Comment:
Default and placeholder do two different things. The former actually sets a default value. The latter specifies what the placeholder text for the value is. So this renders as:

```
  -f, --package-root="."    The directory that contains the package's
                            crossplane.yaml file.
```

It's trying to communicate that the default package-root is the directory `.`.

If it were `placeholder:"PATH"` it would render as:

```
  -f, --package-root=PATH    The directory that contains the package's
                            crossplane.yaml file.
```

This is trying to communicate how to use the flag - that the expected value is a path. It's in all caps to draw attention to the fact that it's an example value.

FWIW this is kinda just how Kong (our CLI library) works.